context/app/static/js/hooks/useCollectionsData.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { getAuthHeader } from 'js/helpers/functions';
+
+function useCollectionsData(elasticsearchEndpoint, nexusToken) {
```

Comment:
Is there a reason we can't have this request from the server side? As with the other entity pages, I want to make sure we don't have soft 404s.

Reply:
This is for all of the collections data not a specific entity. I added it here because eventually we will eventually be moving towards adding collections as a search page and it's easier to put this in temporarily and remove it down the road then passing everything in through flask. Does that make sense @mccalluc?

Reply:
A single collection entity is coming from the server side.

Reply:
That sounds good. Thanks for explaining!